public/index.html

```diff
@@ -27,6 +27,8 @@
     <title>React App</title>
   </head>
   <body>
+    <script src="https://cdn.botpress.cloud/webchat/v0/inject.js"></script>
+    <script src="https://mediafiles.botpress.cloud/a7014ed9-d113-47ea-bdf6-3e63cfc99111/webchat/config.js" defer></script>
```

Comment:
Just wanted to double check this is intentional?

Reply:
yes